ecsmanage/management/commands/ecsmanage.py

```diff
@@ -207,4 +209,8 @@ def run_task(self, config, task_def_arn, security_group_id, subnet_id, cmd):
         #
         # See: https://docs.aws.amazon.com/AmazonECS/latest/userguide/ecs-account-settings.html#ecs-resource-ids  # NOQA
         task_id = task["taskArn"].split("/")[-1]
-        return task_id
+
+        return {
```

Comment:
This might be a nitpick. It seems like `task_id` is read from `task["taskArn"]` anyway, so it might work if we just return `task["taskArn"]` and read the `task_id` in the calling spot of this `run_task` method. Functionality-wise, this makes no difference. So this change is optional and up to you!

Reply:
Great observation! Fixed in da01c53.